src/query/owner/owner.module.ts

```diff
@@ -40,16 +41,12 @@ export class OwnerQueryModule implements OnModuleInit {
     connect(url, { useNewUrlParser: true , useUnifiedTopology: true});
 
     const onEvent = (_, event) => {
-      const modulePrefix = 'owner';
-      const modulePrefixLength = modulePrefix.length;
-      const id = event.streamId.substring(modulePrefixLength + 1, event.streamId.length);
-
       if (event.eventType == EventType.Created) {
```

Comment:
Ha! Very nice!! En als je er nu een `OwnerProcessor` van maakt, dan hoef je hier geen casting te doen. Dan wordt dit:

```ts
this.ownerProcessor(event);
```
Nóg mooier, niet? 😎

Reply:
Hmm... dit werkt in Java prima, maar niet in (uiteindelijk) JavaScript 😏 En toch denk ik wel dat ik de 'casting' in de EventProcessor mooier vind ... (ik doe een commit)